apps/web/src/pages/integrations/components/NovuInAppSetupWarning.tsx

```diff
@@ -0,0 +1,17 @@
+import type { IIntegratedProvider } from '../IntegrationsStoreModal';
+import { SetupWarning } from './SetupWarning';
+import { useInAppActivated } from '../../../api/hooks';
+
+export const NovuInAppSetupWarning = ({ provider }: { provider: IIntegratedProvider | null }) => {
+  const { isInAppActive } = useInAppActivated();
+
+  if (!provider) return null;
+
+  return (
+    <SetupWarning
+      show={!isInAppActive}
+      message="Select a framework to set up credentials to start sending notifications."
+      docReference={provider.docReference}
+    />
+  );
```

Comment:
The Novu In-App setup warning that is shown in the sidebar. It should be hidden when the setup was done properly.